suitcase/spec.py

```diff
@@ -2,20 +2,34 @@
 `Reference <https://github.com/certified-spec/specPy/blob/master/doc/specformat.rst>`_
 for the spec file format.
 """
-import uuid
+import copy
+import logging
 import os
+import uuid
 import warnings
 from datetime import datetime
-import logging
-logger = logging.getLogger(__name__)
 
-import six
-import numpy as np
-import pandas as pd
-import jinja2
 import doct
 import event_model
+import jinja2
 import jsonschema
+import numpy as np
+import pandas as pd
+import six
+
+try:
+    from functools import singledispatch
+except ImportError:
+    # LPy... (╯°□°）╯︵ ┻━┻
```

Comment:
Man, LPy is biting me everywhere... It is now complaining that it doesn't know how to read this file because I don't declare that this is a UTF-8 file. Oh well, guess I'll have to remove the table-flip